app/lib/credentialDisplay/shared/utils/alignment.ts

```diff
@@ -0,0 +1,42 @@
+import { Alignment } from '../../../../types/credential';
+
+export type ValidAlignment = {
+  targetName: string;
+  targetUrl: string;
+  targetDescription?: string;
+};
+
+function isValidUrl(url: string): boolean {
+  try {
+    new URL(url);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+export function getValidAlignments(alignments?: Alignment[]): ValidAlignment[] {
+  if (!alignments || !Array.isArray(alignments)) {
+    return [];
+  }
```

Comment:
If `!Array.isArray(alignments)` is true, alignment can be a single item (and not array).

Reply:
Ah, never mind, I see in the OBv3 JSON Schema, alignments is required to be an array.
All good, then.